### ISSUE TITLE
Preprocessing data and initializing model is slow

### ISSUE THREAD

**Issue by junglegobs** [Spine-project/SpinePeriods.jl#6]
This may be simply due to the having to interact with a database, but these can easily take ~ 2 minutes compared to the 30 seconds to solve the ordering problem. Would be nice to speed this up somehow.

**Comment by manuelma** [comment]
Note that most of SpineOpt is lazy loaded using Requires.jl. That means the first run is slower than normal. Can please you confirm that the second run is also *that* slow?

**Comment by junglegobs** [comment]
Yes it is. I need to investigate more however. I put it here as a placeholder in case someone eventually complains about this.

**Comment by junglegobs** [comment]
In fact now I've noticed it even gets stuck at the preprocessing step. Not sure what this could be.
